src/persist/src/nemesis/direct.rs

```diff
@@ -481,17 +477,19 @@ impl DirectWorker {
     }
 
     fn write_multi(&mut self, req: WriteReqMulti) -> Result<PFuture<SeqNo>, Error> {
-        let mut write_handles = Vec::new();
         let mut updates = Vec::new();
         for req in req.writes {
             let stream = self.stream(&req.stream)?;
-            updates.push((stream.stream_id, vec![req.update]));
-            write_handles.push(stream.write.clone());
+            updates.push((stream.write.clone(), vec![req.update]));
```

Comment:
how odd, this PR seems to be making direct_mem flaky but I don't immediately see why

Reply:
oh! I bet it's removing the `let stream_id = write.stream_id()?;` call above